deployment/helm/values.yaml

```diff
@@ -59,6 +59,17 @@ controller:
   # nodeSelectors is the array of nodeSelectors used to target nodes for the Akri Controller to run on
   # This can be set from the helm command line using `--set controller.nodeSelectors.label="value"`
   nodeSelectors: {}
+  resources:
+    # memoryRequest defines the minimum amount of RAM that must be available to this Pod
+    # for it to be scheduled by the Kubernetes Scheduler
+    memoryRequest: 100Mi
```

Comment:
if we are going to have a universal value, i wonder if it wouldn't make sense to have it declared once ... and have a per-component override.

something like 

```
memoryRequest: 100mi
...
agent:
  resources:
    memoryRequest: 
...
```

then the template can choose .Values.agent.resources.memoryRequest if it is provided ...  otherwise it can choose .Values.memoryRequest.

with that, users could with one setting adjust all the pods to a limit.

Reply:
After reducing the minimums, the values have diverged -- expect for the Discovery Handlers. I updated the [HackMD](https://hackmd.io/@akri/rkmkDJ88O) document and push the latest values. Since they are different, it seems a universal value is not needed/applicable?

Reply:
fair enough :)